components/director/internal/model/formation_assignment.go

```diff
@@ -23,16 +24,18 @@ const (
 
 // FormationAssignment represent structure for FormationAssignment
 type FormationAssignment struct {
-	ID          string                  `json:"id"`
-	FormationID string                  `json:"formation_id"`
-	TenantID    string                  `json:"tenant_id"`
-	Source      string                  `json:"source"`
-	SourceType  FormationAssignmentType `json:"source_type"`
-	Target      string                  `json:"target"`
-	TargetType  FormationAssignmentType `json:"target_type"`
-	State       string                  `json:"state"`
-	Value       json.RawMessage         `json:"value"`
-	Error       json.RawMessage         `json:"error"`
+	ID                            string                  `json:"id"`
+	FormationID                   string                  `json:"formation_id"`
+	TenantID                      string                  `json:"tenant_id"`
+	Source                        string                  `json:"source"`
+	SourceType                    FormationAssignmentType `json:"source_type"`
+	Target                        string                  `json:"target"`
+	TargetType                    FormationAssignmentType `json:"target_type"`
+	State                         string                  `json:"state"`
+	Value                         json.RawMessage         `json:"value"`
+	Error                         json.RawMessage         `json:"error"`
+	LastStateChangeTimestamp      *time.Time
```

Comment:
Why no json tags? Is that intentional? Where do we use model structs for json (un)marshalling?

Reply:
I think these tags are not used directly because we use the converters but I added them for consistency in [this PR](https://github.com/kyma-incubator/compass/pull/3615).